cmake/Modules/opm-material-prereqs.cmake

```diff
@@ -16,8 +16,8 @@ set (opm-material_DEPS
 	# compile with C++0x/11 support if available
 	"CXX11Features REQUIRED"
 	# prerequisite OPM modules
-	"opm-common"
 	"opm-parser"
+	"opm-common"
```

Comment:
And the conclusion is: Both opmparser and opmcommon *should* be marked as `REQUIRED`?